### ISSUE TITLE
Request PIN from device with MiracleCast

### ISSUE THREAD

**Issue by penn5** [albfan/miraclecast#282]
Hello,

My Microsoft Display Adapter is not displaying a PIN code when I connect to it with MiracleCast. As a result the connection fails with error 'Wrong PIN' (on the display adapter)

Is there a way to request that the device displays a PIN code. It is clear that Windows does this to facilitate connection to the Display Adapter.

Thanks!

**Comment by penn5** [comment]
I've got all the steps in a drafted comment, but I didn't get a chance to post it. I'm on my phone now so just wait a bit

**Comment by penn5** [comment]
I have ended wpa_supplicant and NetworkManager, and started miracle-wifictl as root in background.
Then I ran:
`sudo miracle-wifictl`
`select 2`
`p2p-scan`
`connect <devicename>`
No more output is issued for a while. However, the Adapter flickers white then returns to the connection screen.
Then MiracleCast says:
`[FAIL] Peer: 9a:5f:d3:c1:5d:88@2 Reason: group owner negotiation failed`
`[REMOVE] Peer: 9a:5f:d3:c1:5d:88@2`
